components/src/core/components/Pagination/Pagination.vue

```diff
@@ -162,11 +162,11 @@ export default defineComponent({
     },
 
     showPagination() {
-      if ((this.totalRecordsCount <= this.pagesList[0]) && (this.length <= 1)) {
+      if (this.totalRecordsCount <= this.pagesList[0] && this.length <= 1) {
         return false;
       }
       return true;
-    }
+    },
```

Comment:
is this correct?

Reply:
never-mind i was confused as this curly brace is for  the if block

Reply:
Noted